modules/programs/astroid.nix

```
@@ -36,7 +36,7 @@ let
 
   # See https://github.com/astroidmail/astroid/wiki/Configuration-Reference
   finalConfig = let
-    template = lib.fromJSON (lib.readFile ./astroid-config-template.json);
+    template = builtins.fromJSON (lib.readFile ./astroid-config-template.json);
```

Previous comment was deleted as I wasn't sure which `importJSON` is at library root, but it is indeed [`lib.trivial.importJSON`](https://github.com/NixOS/nixpkgs/blob/32a40c2a6bc36c3d6ebd914da27c39e0dba66c67/lib/default.nix#L138) (and not [`lib.modules.importJSON`](https://github.com/NixOS/nixpkgs/blob/32a40c2a6bc36c3d6ebd914da27c39e0dba66c67/lib/modules.nix#L1917)).

```suggestion
    template = lib.importJSON ./astroid-config-template.json;
```